TechArea20221206.md (#40)
osg-htc/technology: docs/meetings/2022/TechArea20221206.md

## OSG Software Team

-   AI (Carl): EL9 support next items
    -   Create 'buildsys-*' packages (SOFTWARE-5391, SOFTWARE-5392)
    -   Build 'fetch-sources' (SOFTWARE-5393)
-   AI (BrianL): NVidia GPU software-base image (SOFTWARE-5368)

### Discussion

-   Time limit on GHA jobs; may want to consider paid tier for github;
    cross that bridge when we get there.

-   Marco: what's the story with condor el9 support?
    -  10.2.0 will have el9 support
    -  10.0 LTS will not get el9 support (for reasons relating to cgroups support)

>     -  el7 EOL in May will trigger updates

Reviewer: ```suggestion
    -  el7 EOL in May 2024 will trigger updates
```

Though HTCondor 9.0 will EOL in May 2023

>     -  until people scream we are not going to add el9 support to 10.0

Reviewer: ```suggestion
```

Let's not declare this publicly :)

### Discussion

-   condor 10.0.0 / with upgrading from 9->10;
    -   with gotchas:
        -  mapfile pcre2 character classes
        -  trust domain used to be derived from condor host;
           if trust domain changes need to re-issue tokens.
           THIS MAY BREAK SITES.
           Sites may not want to re-issue tokens;
           may want to go back to trust domain based on condor host.
        -  Used to specify gpus with requirements section;
           now there's a request-gpu keyword:

>            people will have to change how they're doing this.

Reviewer: ```suggestion
          end  users will have to change how they're doing this.
```
